tests/e2e/tests/02-enable-serve-via-traffic/send_traffic.sh

```
@@ -1,12 +1,126 @@
+        echo "Attempting verbose request for debugging..."
+        kubectl exec -n "$NAMESPACE" "$POD_NAME" -- curl \
+            --max-time 10 \
+            -v \
+            "$URL" 2>&1 | head -20 || echo "  - Verbose request failed"
+        
+        exit 2
+    fi
+    
+    echo "SUCCESS: Request $i completed successfully (HTTP $code in ${duration}s)"
+    
+    if [ $i -lt $MAX_RETRIES ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)